Engine/src/Engine/Physics/OctNode.h

```diff
@@ -0,0 +1,107 @@
+#pragma once
+
+#include <list>
+#include <vector>
+#include "glm/glm.hpp"
+#include "AABB.h"
+
+namespace Engine
+{
+	/// <summary>
+	/// Forward declaration of Physics Component
+	/// </summary>
+	struct PhysicsComponent;
+	/// <summary>
+	/// Forward declaration of Entity
+	/// </summary>
+	class Entity;

```

Comment:
just a suggestion, I think it's a bit more readable if you make one comment for all forward declarations